### PR TITLE
Update code to work on latest version

### PR DESCRIPTION
**Pull request by olehbozhok**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -39,47 +40,33 @@ impl MultiLevelPerceptron {
     }
 }
 
-
 pub fn main() -> anyhow::Result<()> {
     let dev = Device::cuda_if_available(0)?;
 
-    let train_votes_vec: Vec<u32> = vec![
```
Comment (evgenyigumnov) [inline src/main.rs]: Could you restore code formating from 46 to 81 lines?

**Comment by evgenyigumnov** [comment]
Could solve my comment, please?

**Comment by olehbozhok** [comment]
> Could solve my comment, please?
@evgenyigumnov Yes, sure, where I can see your remarks?

File: src/main.rs (hunk 1 above)
Comment (olehbozhok) [reply src/main.rs]: I have done

**Comment by evgenyigumnov** [comment]
[Update code to work on latest version](https://github.com/evgenyigumnov/candle-simplified-example/pull/1#top)

**Pull request merged**